index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="assets/css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <div></div>
+        <h1>Chase Lipscomb</h1>
+        <div class="top">
+            <span>About</span>
+            <span id="two">Portfolio</span>
```

Comment:
In here you can add an <a href="nfjdsfds"> and the link would be to other html pages so you can navigate to them